### PR TITLE
Fix webSocketProxy.js

### PR DESCRIPTION
**Pull request by martinbigio**
cc @mkonicek @ide

### CODE REVIEW

File: packager/webSocketProxy.js [hunk 1]
```diff
@@ -59,11 +59,12 @@ function attachToServer(server, path) {
 
   return {
     server: wss,
-    isChromeConnected: () =>
-      clients
-        .map(ws => ws.upgradeReq.headers['user-agent'])
+    isChromeConnected: function(){
```
Comment (ide) [inline packager/webSocketProxy.js]: nit: space between parens and curly

**Comment by ide** [comment]
LGTM -- but you'll have to merge this on the fb side right?

(We didn't cut the 0.14 branch yet btw)

**Comment by martinbigio** [comment]
Oh, great, lets get it in then :). Just fixed the nit

**Comment by martinbigio** [comment]
@facebook-github-bot shipit

**Comment by martinbigio** [comment]
I wonder if the bot will say I'm a cheater :D

**Comment by frantic** [comment]
@martinbigio what was the issue here? I assumed by this point we already have `babel/register`, no?

**Comment by martinbigio** [comment]
@frantic we do but babel didn't have the directory `packager` (where `webSocketProxy.js`) was whitelisted. I'm moving a lot of the stuff from `packager` to `local-cli` because most of it it's middleware and `server` dependencies :)

**Pull request closed without merging**